src/components/sections/FindPokemon/FindPokemon.module.scss

```diff
@@ -0,0 +1,42 @@
+.container {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    width: 100%;
+    height: 420px;
+    margin: 20px;
+    background-color: #342547;
+
+    &__search {
+
+        & p {
+            font-size: 48px;
+            font-weight: bold;
+            color: #FFFFFF;
+            text-align: center;
+        }
+    }
+
+
+    @media screen and (max-width: 870px) {
+       & img {
+        width: 350px;
+        height: 300px;
```

Comment:
Check the indentation you are using (Are you using 2 spaces or 4 spaces?) The code formatter can also help you with this